crates/agglayer-config/src/lib.rs

```diff
@@ -162,6 +166,7 @@ impl Config {
             prover_entrypoint: default_prover_entrypoint(),
             prover: Default::default(),
             debug_mode: false,
+            mock_verifier: false,
```

Comment:
It would be nicer for this to be a field like `verifier` of the type `enum` with the `Cpu` and `Mock` variants. Would show up as `verifier = "cpu"` or `verifier = "mock"` in the config.

Reply:
I first try this, but end up using a flag, as this should not be used to configure the verifier, just turn on/off the mock for tests. We'll not have something like `verifier = "gpu"` or `verifier = "network"`, at least for now. I'll stick with this format, and if we do want to configure the `verifier` at some point, we'll be able to and we'll deprecate this flag if needed.